daisi/src/cpps/logical/amr/amr_logical_agent.cpp

```diff
@@ -138,8 +138,8 @@ void AmrLogicalAgent::sendTopologyToPhysical() {
 void AmrLogicalAgent::sendTaskToPhysical() {
   material_flow::Task task = order_management_->getCurrentTask();
 
-  daisi::cpps::mrta::model::Ability ability(
-      42.0, daisi::cpps::mrta::model::LoadCarrier::kNoLoadCarrierType);
+  amr::AmrStaticAbility ability(amr::LoadCarrier(amr::LoadCarrier::Types::kNoLoadCarrierType),
+                                42.0);
```

Comment:
Maybe there should be a constant definition for the default max_payload_weight_kg?